frontend/src/components/ProfModal/OverviewPanel/OverviewPanel.tsx

```diff
@@ -182,7 +183,7 @@ function SeasonRatingChart({
   const chartData: ChartData<'line', { x: number; y: number }[]> = {
     datasets: [
       {
-        label: 'Average rating',
+        label: 'Average Rating',
```

Comment:
I've changed us to use sentence case consistently, because our font looks really terrible when combined with title case (looks too bumpy)